packages/features/schedules/lib/use-schedule/useSchedule.ts

```diff
@@ -36,17 +41,14 @@ export const useSchedule = ({
       // if `prefetchNextMonth` is true, two months are fetched at once.
       endTime: (prefetchNextMonth ? nextMonthDayjs : monthDayjs).endOf("month").toISOString(),
       timeZone: timezone!,
-      eventTypeId: eventId!,
       duration: duration ? `${duration}` : undefined,
+      eventTypeId: eventId,
+      org,
+      isTeamEvent,
```

Comment:
we need either eventTypeId or org, isTeamEvent, username and eventTypeSlug to get the eventTypeId on the server side